scripts/hotfixes/kdump/create-kdump-artifacts.sh

```diff
@@ -0,0 +1,256 @@
+#!/bin/bash
+#
+# MIT License
+#
+# (C) Copyright 2021-2022 Hewlett Packard Enterprise Development LP
```

Comment:
```suggestion
# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
```

Reply:
It is copied from another repository that has had this script since 2021.

Reply:
The PR for this script is here: https://github.com/Cray-HPE/metal-provision/pull/12

NOTE that this script omits 3 more dracut modules than what is being committed in that metal-provision PR.